OpenOversight/app/utils.py

```diff
@@ -1,11 +1,30 @@
+import boto3
 import config
 import datetime
 from flask import current_app
 from flask_sqlalchemy import SQLAlchemy
+import hashlib
 from sqlalchemy import desc, asc, func
 from sqlalchemy.sql.expression import cast
 from .models import db, Officer, Assignment, Image, Face
-import pdb
+
+
+def hash_file(file_to_hash):
+    return hashlib.sha256(file_to_hash).hexdigest()
+
+
+def upload_file(src_filename, dest_filename):
+    s3_client = boto3.client('s3')
+    s3_client.upload_file('/tmp/{}'.format(src_filename),
+                          current_app.config['S3_BUCKET_NAME'],
+                          dest_filename,
```

Comment:
(loading the s3 web console is a terrible experience with 150k files in the same dir too, trust me)